src/components/toggle/styled.tsx

```diff
@@ -0,0 +1,86 @@
+import React from "react"
+import styled from "styled-components"
+import { getColor, getSizeUnit } from "../../theme/utils"
+import { controlFocused } from "../../mixins"
+
+export const ToggleContainer = styled.div`
+  display: block;
+  box-sizing: border-box;
+  width: 40px;
+  height: 20px;
+`
+
+export const HiddenToggleInput = styled.input.attrs({
+  type: "checkbox",
+})`
+  display: none;
+`
+
+export const StyledToggle = styled.div<{
+  checked: boolean
+  disabled?: boolean
+  colored?: boolean
+}>`
+  box-sizing: border-box;
+  width: 40px;
+  height: 20px;
+  background: ${getColor(["white", "pure"])};
+  border: 1px solid ${getColor(["gray", "bombay"])};
+  border-radius: 100px;
+  transition: all 150ms;
+
+  display: block;
+  position: relative;
+
+  -webkit-tap-highlight-color: transparent;
+  flex-shrink: 0;
+  align-self: flex-start;
+  cursor: pointer;
+
+  &:after {
+    display: block;
+    position: absolute;
+    content: "";
+    width: 16px;
+    height: 16px;
+    border-radius: 50%;
+    left: 5%;
+    top: 50%;
+    transform: translateY(-50%);
+    transition: left 0.2s ease;
+    background-color: ${({ disabled, colored, checked }) => {
+      if (disabled) return getColor(["gray", "gainsboro"])
```

Comment:
If you want to use explicit call of `getColor` you have to provide second call with `theme`

This implementation is about this usage:
```typescript
// implicit way
color: ${getColor(["gray", "gainsboro"])};
// explicit way
color: ${({theme, ...props}) => getColor(["gray", "gainsboro"])({theme})};
```